src/v0/destinations/zendesk/transform.js

```diff
@@ -336,18 +340,28 @@ async function createUser(message, headers, destinationConfig, baseEndpoint, typ
   const payload = { user: userObject };
 
   try {
-    const resp = await myAxios.post(url, payload, config, {
-      destType: 'zendesk',
-      feature: 'transformation',
-    });
+    const { processedResponse: processedResponseZn } = await handleHttpRequest(
+      'post',
+      url,
+      payload,
+      config,
+      {
+        destType: 'zendesk',
+        feature: 'transformation',
+      },
+    );
 
-    if (!resp.data || !resp.data.user || !resp.data.user.id) {
+    if (
+      !processedResponseZn.response ||
```

Comment:
I think just `!processedResponseZn?.response?.user?.id` should be enough.

Reply:
done